islands/Chart.tsx

```diff
@@ -84,14 +84,9 @@ export default function Chart<
   }, []);
 
   return (
-    <div
-      {...container}
-      ref={containerRef}
```

Comment:
**suggestion:** as we don't have a container anymore i would make sense to remove, `container` from `ChartProps` and `containerRef` from `useChart`.
and as `useResizeObserver` has no truthy target anymore, we could remove the hook as well.